src/touch.js

```diff
@@ -0,0 +1,89 @@
+var common = require('./common');
+var fs = require('fs');
+
+module.exports = _touch;
+
+//@
+//@ ### touch(file)
+//@
+//@ Examples:
+//@
+//@ ```javascript
+//@ touch('file.txt');
+//@ ```
+//@
+//@ Creates a file if it does not exist. Sets the file's mtime and ctime to NOW if the file
+//@ does exist
+function _touch(opts, file) {
+	opts = common.parseOptions(opts, {
+		'a': 'atime',
+		'c': 'no_create',
+		'd': 'date',
+		'm': 'mtime',
+		'r': 'reference',
+	});
+
+	if (!file) {
+		common.error('no paths given');
+	}
+
+	if (typeof file !== 'string') {
```

Comment:
Could we allow this to support a `file` argument that is either a string or an array of strings? Similar to how [`ls()`](https://github.com/shelljs/shelljs/blob/master/src/ls.js#L41) does it.